generate-moz-grammars/generate-mozcpp.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# This script updates the mozcpp grammar automatically.
+#
+# Usage: ./generate-moz-grammars/generate-mozcpp.sh
+
+# FIXME we need to remove this line once we are going to use
+# the tree-sitter-cpp bindings
+# Update tree-sitter-cpp submodule such that is the same one used within the
+# tree-sitter-mozcpp directory
+git submodule update --remote tree-sitter-cpp
```

Comment:
Instead of updating the tree-sitter-cpp submodule as part of this, we can just get its revision and, in the tree-sitter-mozcpp directory, clone the repo at that revision

Reply:
Good idea! :smiley: